Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd/loader.go

```diff
@@ -35,8 +35,13 @@ const (
 	RecommendedConfigPathEnvVar = "KUBECONFIG"
 )
 
-// ClientConfigLoadingRules is a struct that calls our specific locations that are used for merging together a Config
+// ClientConfigLoadingRules is a collection that allows multiple sets of loading rules
 type ClientConfigLoadingRules struct {
+	Rules []ClientConfigLoadingRule
```

Comment:
I think this would be cleaner as a simple string slice that is required for NewClientConfigLoadingRules(filepriority []string).